polygon_cleanup_tracks.md

```diff
@@ -0,0 +1,31 @@
+## Cleanup tracks based on spatial relationship with a polygon feature layer
+
+This script allows cleanup of track points from a tracks layer based on a spatial relationship to polygon geometry. You provide the script with a feature layer containing polygons, whether or not you want to delete track points inside or outside the polygons, and the script will perform a spatial intersection and delete any necessary features.
+
+By default, this script will delete the tracks found inside the polygons. However, by providing the --symmetric-difference parameter, the use can delete tracks falling outside the polygons and preserve the ones inside.
+
+Location Tracking must be enabled for your organization to use this script. You must be an admin to use this script. 
+
+Supports Python 3.6+
+
+----
+
+Other than the authentication arguments (username, password, org) the script uses the following parameters:
+
+- -layer-url <layer_url> - The feature service URL for your polygon feature layer with features that will be spatially intersected with track points. This is required.
+- -where <where_clause> - The where clause used to filter out only certain polygons for the spatial comparison (for example, OBJECTID > 1). Defaults to 1=1 (all features are used for comparison)
+- --symmetric-difference <symmetric_difference> - A parameter when provided, deletes features that fall outside the polygons. If not provided, delete track points inside the polygons. 
+
+Example Usage 1
+```python
```

Comment:
should be bash since it's a command line example

Reply:
👍